Description: Disable the notification preferences so options are not presented that indicator-power handles.
Author: Martin Wimpress <code@flexion.org>

--- a/data/gpm-prefs.ui
+++ b/data/gpm-prefs.ui
@@ -895,7 +895,7 @@
                 </child>
                 <child>
                   <object class="GtkVBox" id="vbox_notification">
-                    <property name="visible">True</property>
+                    <property name="visible">False</property>
                     <property name="orientation">vertical</property>
                     <property name="spacing">6</property>
                     <child>
